modules/backend/behaviors/listcontroller/_list_toolbar.php

```diff
@@ -0,0 +1,26 @@
+<?php
+$listController = $this->getClassExtension(\Backend\Behaviors\ListController::class);
+$listConfig = $listController->getConfig();
+?>
+
+<div data-control="toolbar">
+    <a
+        href="<?= url()->current() . '/create' ?>"
+        class="btn btn-primary wn-icon-plus">
+        <?= e(trans('backend::lang.form.create_title', ['name' => trans(\Winter\Storm\Support\Str::before($listConfig->title, '_plural'))])); ?>
+    </a>
+
+    <button
+        class="btn btn-danger wn-icon-trash-o"
+        disabled="disabled"
+        onclick="$(this).data('request-data', { checked: $('.control-list').listWidget('getChecked') })"
+        data-request="onDelete"
+        data-request-confirm="<?= e(trans('backend::lang.list.delete_selected_confirm')); ?>"
+        data-trigger-action="enable"
+        data-trigger=".control-list input[type=checkbox]"
+        data-trigger-condition="checked"
+        data-request-success="$(this).prop('disabled', 'disabled')"
+        data-stripe-load-indicator>
+        <?= e(trans('backend::lang.list.delete_selected')); ?>
+    </button>
```

Comment:
```suggestion
    </button>
    <?= $this->fireViewEvent('backend.list.extendListToolbar') ?>
```

Reply:
@mjauvin I'd prefer if the list toolbar buttons were configurable more in the way that the relationcontroller handles buttons instead of a generic view event.

Reply:
You can also just add `_list_toolbar.php` to your controller's folder and it'll be overwritten